### PR TITLE
fix(@toss/date): unify rules

### PR DESCRIPTION
**Pull request by SEOKKAMONI**
## Overview

The rules were unified

**question: Is there a reason why only certain parts are arrow functions?**

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/common/date/src/date.ts [hunk 1]
```diff
@@ -8,10 +8,10 @@ export type DateFnsDateType = number | Date;
 
 export const kstFormat = (date: DateFnsDateType, format: string) => _format(date, format, { locale });
 
-export const roundUpHoursInDays = (hours: number) => {
+export function roundUpHoursInDays(hours: number) {
```
Comment (minsoo-web) [inline packages/common/date/src/date.ts:11]: I don't think it's a big issue, but if unifying is the main thing in this PR, shouldn't `kstFormat` be changed as well?

File: packages/common/date/src/date.ts [hunk 2]
```diff
@@ -8,10 +8,10 @@ export type DateFnsDateType = number | Date;
...
   const remainder = hours % 24;
   return remainder === 0 ? hours : hours + 24 - (hours % 24);
-};
+}
```
Comment (minsoo-web) [inline packages/common/date/src/date.ts:14]: `"semi": true,` is defined in prettierrc file.

File: packages/common/date/src/date.ts (hunk 1 above)
Comment (SEOKKAMONI) [reply packages/common/date/src/date.ts:11]: I agree that this is not a big issue but Wouldn't convention be important at work? That's why I posted it

I think I missed that part

**Comment by SEOKKAMONI** [comment]
I don't think it's that important, so I'll close it sorry..

**Pull request closed without merging**